tests/EdFi.Ods.Api.IntegrationTests/ParentUSIToEducationOrganizationIdAuthViewTests.cs

```diff
@@ -0,0 +1,284 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using NUnit.Framework;
+using System;
+
+namespace EdFi.Ods.Api.IntegrationTests
+{
+    [TestFixture]
+    public class ParentUSIToEducationOrganizationIdAuthViewTests : DatabaseTestFixtureBase
+    {
+        [Test]
+        public void When_parent_assigned_to_a_student_in_school_that_belongs_to_a_district_should_not_return_duplicate_records()
+        {
+            var studentUniqueId = Guid.NewGuid().ToString("N");
+            var parentUniqueId = Guid.NewGuid().ToString("N");
+
+            Builder
+                .AddLocalEducationAgency(5601)
+                .AddSchool(9805, 5601).AddStudent(studentUniqueId).AddParent(parentUniqueId)
+                .Execute();
+
+            var studentUSI = AuthorizationViewHelper.GetStudentUSI(Connection, studentUniqueId);
```

Comment:
For all uses of USI, the casing should be "Usi" to reflect coding standards.

Reply:
I fixed it